pkg/services/ngalert/notifier/config.go

```diff
@@ -14,7 +14,7 @@ import (
 	api "github.com/grafana/grafana/pkg/services/ngalert/api/tooling/definitions"
 )
 
-var cfglogger = log.New("notifier.config")
+var cfglogger = log.New("ngalert.notifier.config")
```

Comment:
Let's remove it at all and update `PersistTemplates` to get logger as an argument. Reason: the logger will contain some context like orgID.

Reply:
I updated it

Reply:
great. Can you remove `cfglogger`?